api/client/service/scale.go

```diff
@@ -60,13 +60,27 @@ func runScale(dockerCli *client.DockerCli, args []string) error {
 func runServiceScale(dockerCli *client.DockerCli, serviceID string, scale string) error {
 	client := dockerCli.Client()
 	ctx := context.Background()
+	headers := map[string][]string{}
 
 	service, _, err := client.ServiceInspectWithRaw(ctx, serviceID)
 
 	if err != nil {
 		return err
 	}
 
+	// TODO(nishanttotla): Is this the best way to get the image?
+	image := service.Spec.TaskTemplate.ContainerSpec.Image
```

Comment:
Remember you can change the image as part of an update (potentially from a different registry).